diff --git a/Tinycast/Migration/StorageRelocation.swift b/Tinycast/Migration/StorageRelocation.swift
index 909a140..b25d316 100644
--- a/Tinycast/Migration/StorageRelocation.swift
+++ b/Tinycast/Migration/StorageRelocation.swift
@@ -10,7 +10,7 @@ enum StorageRelocation {
     static let removeAfter =
         DateComponents(
             calendar: Calendar(identifier: .gregorian), timeZone: .gmt,
-            year: 2026, month: 9, day: 5
+            year: 2026, month: 12, day: 31
         ).date ?? .distantPast
 
     private static let databaseName = "clipboard.sqlite3"
